### PR TITLE
nginxModules.video-thumbextractor: update to unstable and switch to ffmpeg_4

### PR DESCRIPTION
**Pull request by zaninime**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/http/nginx/modules.nix [hunk 1]
```diff
@@ -488,10 +488,10 @@ in
       name = "video-thumbextractor";
       owner = "wandenberg";
       repo = "nginx-video-thumbextractor-module";
-      rev = "0.9.0";
-      sha256 = "1b0v471mzbcys73pzr7gpvzzhff0cva0l5ff32cv7z1v9c0ypji7";
+      rev = "92b80642538eec4cfc98114dec5917b8d820e912";
```
Comment (SuperSandro2000) [inline pkgs/servers/http/nginx/modules.nix:491]: Why did you change that?

Comment (zaninime) [reply pkgs/servers/http/nginx/modules.nix:491]: I forgot to report what I wrote on the issue about ffmpeg 3: https://github.com/NixOS/nixpkgs/issues/120705#issuecomment-845820604

File: pkgs/servers/http/nginx/modules.nix [hunk 2]
```diff
@@ -488,10 +488,10 @@ in
...
+      sha256 = "0a8d9ifryhhnll7k7jcsf9frshk5yhpsgz7zgxdmw81wbz5hxklc";
     };
-    inputs = [ pkgs.ffmpeg_3 ];
+    inputs = [ pkgs.ffmpeg_4 ];
```
Comment (dotlambda) [inline pkgs/servers/http/nginx/modules.nix:494]: Why not
```suggestion
    inputs = [ pkgs.ffmpeg ];
```
?

**Comment by dotlambda** [comment]
@zaninime How do I test wether this builds?

**Review by jonringer — APPROVED** [review]
diff LGTM

would like for someone to test this scenario

**Comment by zaninime** [comment]
@dotlambda @jonringer you can add `nginxModules.thumbextractor-video` to either `nginxMainline` or `nginxStable` in `all-packages.nix` and build it.

**Comment by jonringer** [comment]
I would like some confirmation that someone tried to use it. My concern is that it's packaged using a thin wrapper and defers correctness to runtime, and having a successful build doesn't mean much.

**Comment by zaninime** [comment]
I can test it from Tuesday onwards.

**Review by thiagokokada — APPROVED** [review]
LGTM.

I am going to merge this.

The commit this one points explicitly recommends FFMPEG 4 (https://github.com/wandenberg/nginx-video-thumbextractor-module/commit/92b80642538eec4cfc98114dec5917b8d820e912), and if this breaks something I think someone can open an issue later once this lands on `master`.

As it if right now, this is a possible security issue (specially in something as `nginx`), and nobody seems to have the structure to test it locally.
